packages/frontend/src/components/MarkdownRenderer/index.tsx

```diff
@@ -27,7 +27,7 @@ interface MarkdownRendererProps {
 }
 
 export default function MarkdownRenderer(props: MarkdownRendererProps) {
-  const { allowMultilineBreaks = true, source } = props
+  const { allowMultilineBreaks = true, source, components = {} } = props
```

Comment:
wah nice u helped me to add back the customizability part thks

Reply:
anything for you